### ISSUE TITLE
`indent`: `RangeError: Invalid count value: -3` for multiline export declaration with `VariableDeclarator` to `first`

### ISSUE THREAD

**Issue by lwr** [eslint-stylistic/eslint-stylistic#1208]
### Validations

- [x] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

## Minimal example or steps to reproduce the bug?
config
```js
{
    rules: {
        '@stylistic/indent': ['error', 4, {
            VariableDeclarator  : 'first',
        }]
    }
}
```

sample.js
```javascript
export const
    CONST_1    = 1,
    CONST_2    = 2,
    CONST_3    = 3,
    CONST_LAST = null;
```

## What happened?
```
Oops! Something went wrong! :(

ESLint: 10.3.0

RangeError: Invalid count value: -3
Rule: "@stylistic/indent"
    at String.repeat (<anonymous>)
    at OffsetStorage.getDesiredIndent (file:///path/to/node_modules/@stylistic/eslint-plugin/dist/rules/indent.js:244:123)
    at Program:exit (file:///path/to/node_modules/@stylistic/eslint-plugin/dist/rules/indent.js:1134:56)
```

## Environment
- ESLint 10.3.0
- `@stylistic/eslint-plugin` 5.10.0

## Workarounds
```json
{
    "rules": {
        "@stylistic/indent": ["error", 4, {
            "ignoredNodes" : ["ExportNamedDeclaration > VariableDeclaration > VariableDeclarator"]
        }]
    }
}
```

### Reproduction

https://stackblitz.com/edit/github-xiqkv9sg

### System Info

```shell
System:
    OS: macOS 26.4.1
    CPU: (10) arm64 Apple M1 Max
    Memory: 1.48 GB / 64.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 25.9.0 - /opt/homebrew/bin/node
    npm: 11.12.1 - /opt/homebrew/bin/npm
    bun: 1.3.13 - /opt/homebrew/bin/bun
```

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests
